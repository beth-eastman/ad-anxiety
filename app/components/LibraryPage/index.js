/**
 * LibraryPage contains the list of sections for library content. The List
 * will provide links for the user to view content on that secition.
 *
 * Bethany Eastman
 */

import React from 'react';
import { Card, List, ListItem } from 'material-ui';
import { browserHistory } from 'react-router';
import { library } from 'data/library';

export default class LibraryPage extends React.Component {      // eslint-disable-line

  /* Change the title */
  componentWillMount() {
    this.props.setPageTitle("Library"); // eslint-disable-line
    this.props.changeIcon('Home'); // eslint-disable-line
  }

  /*
   * Iterate over the list of sections and render a clickable list item
   * linking to the related content
   */
  render() {
    return (
      <div className="librarypage">
        <Card>
          <List>
            {library.map((section) => (
              <ListItem
                key={section.key}
                primaryText={section.title}
                onTouchTap={() => {
                  browserHistory.push('/library/'.concat(section.key));
                }}
              />
            ))}
          </List>
        </Card>
      </div>
    );
  }
}
