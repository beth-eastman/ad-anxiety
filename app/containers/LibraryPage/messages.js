/*
 * Library Page Messages
 *
 * This contains all the text for the Library Page component.
 */
import { defineMessages } from 'react-intl';

export default defineMessages({
  sectionOne: {
    id: 'app.components.LibraryPage.sectionOne',
    defaultMessage: '1.0 - Introduction to Anxiety',
  },
  sectionTwo: {
    id: 'app.components.LibraryPage.sectionTwo',
    defaultMessage: '2.0 - Anxiety Disorders',
  },
  sectionThree: {
    id: 'app.components.LibraryPage.sectionThree',
    defaultMessage: '3.0 - Other Problems Associated with Anxiety',
  },
  sectionFour: {
    id: 'app.components.LibraryPage.sectionFour',
    defaultMessage: '4.0 - Causes of Anxiety',
  },
  sectionFive: {
    id: 'app.components.LibraryPage.sectionFive',
    defaultMessage: '5.0 - Factors that Make Anxiety Worse',
  },
  sectionSix: {
    id: 'app.components.LibraryPage.sectionSix',
    defaultMessage: '6.0 - Understanding Panic Disorder',
  },
  sectionSeven: {
    id: 'app.components.LibraryPage.sectionSeven',
    defaultMessage: '7.0 - Anxiety Affects Family Members Too',
  },
  sectionEight: {
    id: 'app.components.LibraryPage.sectionEight',
    defaultMessage: '8.0 - How to Manage Anxiety and Worry',
  },
  sectionNine: {
    id: 'app.components.LibraryPage.sectionNine',
    defaultMessage: '9.0 - Getting Support for Anxiety',
  },
  sectionTen: {
    id: 'app.components.LibraryPage.sectionTen',
    defaultMessage: '10.0 - References on Anxiety',
  },
});
