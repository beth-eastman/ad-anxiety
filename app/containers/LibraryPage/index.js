/**
 * Contains library content of the application.
 */

import React from 'react';
import { Card, CardHeader, List, ListItem } from 'material-ui';
import { FormattedMessage } from 'react-intl';
import { browserHistory } from 'react-router';
import messages from './messages';

export default class LibraryPage extends React.Component {      // eslint-disable-line

  componentWillMount() {
    this.props.setPageTitle("Library"); // eslint-disable-line
  }

  render() {
    return (
      <div className="librarypage">
        <Card>
          <CardHeader
            title="Library"
          />
          <List>
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionOne} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionTwo} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionThree} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionFour} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionFive} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionSix} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionSeven} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionEight} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionNine} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
            <ListItem
              primaryText={<FormattedMessage {...messages.sectionTen} />}
              onTouchTap={() => {
                browserHistory.push('/library/page1');
              }}
            />
          </List>
        </Card>
      </div>
    );
  }
}
