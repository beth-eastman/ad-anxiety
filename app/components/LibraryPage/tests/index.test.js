/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

import LibraryPage from '../index';

it('renders LibraryPage', () => {
  const tree = renderer.create(
    <MuiThemeProvider>
      <LibraryPage params={{ factId: 0 }} setPageTitle={function () {}} changeIcon={function () {}} />
    </MuiThemeProvider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
