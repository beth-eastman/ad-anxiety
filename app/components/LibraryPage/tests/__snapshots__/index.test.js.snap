exports[`test renders LibraryPage 1`] = `
<div
  className="librarypage">
  <div
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0,0,0,0)",
        "backgroundColor": "#ffffff",
        "borderRadius": 2,
        "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
               0 1px 4px rgba(0, 0, 0, 0.12)",
        "boxSizing": "border-box",
        "color": "rgba(0, 0, 0, 0.87)",
        "fontFamily": "Roboto, sans-serif",
        "muiPrepared": true,
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "zIndex": 1,
      }
    }>
    <div
      style={
        Object {
          "paddingBottom": 0,
        }
      }>
      <div
        style={
          Object {
            "muiPrepared": true,
            "padding": "8px 0px 8px 0px",
          }
        }>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  1.0 - Introduction to Anxiety
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  2.0 - Anxiety Disorders
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  3.0 - Other Problems Associated with Anxiety
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  4.0 - Causes of Anxiety
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  5.0 - Factors that Make Anxiety Worse
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  6.0 - Understanding Panic Disorder
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  7.0 - Anxiety Affects Family Members Too
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  8.0 - How to Manage Anxiety and Worry
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  9.0 - Getting Support for Anxiety
                </div>
              </div>
            </div>
          </span>
        </div>
        <div>
          <span
            disabled={undefined}
            href={undefined}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            open={null}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "backgroundColor": null,
                "border": 10,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 16,
                "fontWeight": "inherit",
                "lineHeight": "16px",
                "margin": 0,
                "muiPrepared": true,
                "outline": "none",
                "padding": 0,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "zIndex": 1,
              }
            }
            tabIndex={0}>
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}>
              <div
                style={
                  Object {
                    "marginLeft": 0,
                    "muiPrepared": true,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "paddingTop": 16,
                    "position": "relative",
                  }
                }>
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }>
                  10.0 - References on Anxiety
                </div>
              </div>
            </div>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
